Tools/Scripts/generate-static-analysis-archive

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
```

Comment:
Can we update this script to use `python3` as well?  If not, we can do that later.